### ISSUE TITLE
Support LUA scripting

### ISSUE THREAD

**Issue by ThiefMaster** [jamesls/fakeredis#25]
It would be nice if fakeredis supported LUA scripts like redis does. [LUA bindings for python](http://labix.org/lunatic-python) are available so it might not be that hard - never used them though.

Why would it be useful? Well, chances are good that the usage of lua scripts means there is some more complex logic involved that cannot be easily done with simply redis commands. Chances are good that this logic should be well-tested and thus having lua support in fakeredis would be helpful.

**Comment by bmerry** [comment]
fakeredis 0.10.1 has some scripting support (`eval` command), but I'm leaving this open because it's still incomplete.

**Comment by advance512** [comment]
FYI all, to get Lua support, you need to install it like:

`pip install fakeredis[lua]`

Documentation should be updated..

**Comment by bmerry** [comment]
Good point. I've updated the README on the 1.0 branch. It'll appear in the README when I release 1.0, which should be quite soon.

**Comment by advance512** [comment]
Great, waiting for it. This has been a good drop-in replacement for `mockredis` which is abandoned, or so it seems.

https://github.com/locationlabs/mockredis/issues/130

**Comment by SebastiaanZ** [comment]
I've opened a PR for `SCRIPT EXISTS` and `SCRIPT FLUSH`. Not sure if the implementation is up to standards, but it's here: #281.

**Comment by rodriguez-facundo** [comment]
cjson is not supported :( 

```
Error running script (call to f_7cfc597423ec5ba5cf79190e397fb9dbeaf35226): @user_script:?: [string "<python>"]:139: attempt to index a nil value (global 'cjson')
```
